librubyfmt/src/line_tokens.rs

```diff
@@ -170,6 +181,7 @@ impl ConcreteLineTokenAndTargets {
                 .fold("".to_string(), |accum, tok| {
                     format!("{}{}", accum, tok.into_ruby())
                 }),
+            Self::HeredocString(_hs) => "".to_string(),
```

Comment:
This is technically correct but kind of a kludge. This would need to be revisited after more stuff was finalized.